src/Stripe.net/Infrastructure/Public/StripeConfiguration.cs

```diff
@@ -19,6 +19,12 @@ static StripeConfiguration()
             StripeNetVersion = new AssemblyName(typeof(Requestor).GetTypeInfo().Assembly.FullName).Version.ToString(3);
         }
 
+        public static string StripeApiVersion
+        {
+            get { return stripeApiVersion; }
+            set { stripeApiVersion = value; }
```

Comment:
Does that mean anyone can set this?

Reply:
Yep. I also have doubts whether this should be allowed, but wanted to maintain the current behavior.